crates/matrix-sdk-crypto/src/gossiping/mod.rs

```diff
@@ -91,20 +89,22 @@ impl SecretInfo {
         match &self {
             #[allow(deprecated)]
```

Comment:
No longer needed, right?

Reply:
Yep, removed all of them in https://github.com/matrix-org/matrix-rust-sdk/pull/990/commits/30cc6d23cd5acbbc83b70b65d8b494c85f2bbfd6.